dev/k8s/dpf-sim-controller/Makefile

```
@@ -0,0 +1,84 @@
+	@if [ -n "$(PULL_SECRET)" ]; then \
+		echo "patching imagePullSecrets=$(PULL_SECRET)"; \
+		kubectl -n $(DPF_NAMESPACE) patch deployment dpf-sim-controller --type merge \
+			-p '{"spec":{"template":{"spec":{"imagePullSecrets":[{"name":"$(PULL_SECRET)"}]}}}}'; \
+	fi
+	# Mutable tags (the :dev default) leave a same-tag repush running old bytes;
+	# with imagePullPolicy: Always, a restart forces the fresh pull either way.
+	kubectl -n $(DPF_NAMESPACE) rollout restart deployment/dpf-sim-controller
+	kubectl -n $(DPF_NAMESPACE) rollout status deployment/dpf-sim-controller --timeout=120s
+
+.PHONY: undeploy
+undeploy: ## Remove the simulator (leaves the DPF CRDs in place)
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)